packages/adapter-node/src/index.js

```diff
@@ -1,15 +1,38 @@
 import { handler } from 'HANDLER';
 import { env } from 'ENV';
 import polka from 'polka';
+import http from 'node:http';
+import https from 'node:https';
+import http2 from 'node:http2';
+import fs from 'node:fs';
 
 export const path = env('SOCKET_PATH', false);
 export const host = env('HOST', '0.0.0.0');
 export const port = env('PORT', !path && '3000');
+export const cert_path = env('CERT_PATH', false);
+export const cert_key_path = env('CERT_KEY_PATH', false);
+export const https_port = env('HTTPS_PORT', !path && '3001');
+export const only_https = env('ONLY_HTTPS', false);
+export const no_http2 = env('NO_HTTP2', false);
 
-const server = polka().use(handler);
+const app = polka().use(handler);
```

Comment:
Is there a need to rename this variable? We only end up re-exporting it as "server" later on

Reply:
@ghostdevv Re-exporting as `server` was just so that we don't introduce a breaking change.
The reason I changed it from `server` to `app` was to distinguish it from the "actual" servers that we later create with `http*.createServer`.

Reply:
That makes sense!